### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by rjbullough**
Homework for Tuesday 15 September

### CODE REVIEW

**Comment by rjbullough** [comment]
How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible;
**5**
Was there anything that you struggled with?;
No
Is there anything that you'd like some further information on?;
No
Roughly how long did it take?
60 mins?

File: Ryan Bullough /week01/Tuesday/calculator/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Calculator</title>
+</head>
+<body>
+    <h1>Tuesday 15 Sept</h1>
+    <script src="main.js"></script>
```
Comment (CamillaCdC) [inline Ryan Bullough /week01/Tuesday/calculator/index.html:10]: Usually put the <script> tag inside the <head> tags.

File: Ryan Bullough /week01/Tuesday/calculator/main.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const squareNumber = (a) => {
```
Comment (CamillaCdC) [inline Ryan Bullough /week01/Tuesday/calculator/main.js:1]: love the use of arrow functions!

**Comment by CamillaCdC** [comment]
Nice work!

**Pull request merged**